yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
+    definition: Lattice translation vectors lengths (a, b, c) of an atomic unit cell
+    shape: [3]
+    type: double
+
+  LATTICE_UC_ANGLES:
+    definition: Alpha, Beta and Gamma angles of an atomic unit cell
+    shape: [3]
+    type: double
+
+  SYMMETRY_GROUP:
+    definition: Symmetry group number acording to the Volume A of International Tables for Crystallography
+    shape: [1]
+    type: integer
+
+  CRYSTAL_ORIENTATION_1:
+    definition: Crystallographic orientation defined by Miller Index of the reference surface
+    shape: [3, 3]
+    type: double
+
+  CRYSTAL_ORIENTATION_2:
+    definition: Crystallographic orientation defined by Miller Index of the non-reference surface
+    shape: [3, 3]
+    type: double
+
+  SHAPE_ORIENTATION_1:
+    definition: Real space orientation of a component. Shape-center system reference set to global cartesian system reference.
+    shape: [1, 3]
+    type: double
+
+  SHAPE_ORIENTATION_2:
+    definition: Real space orientation of a component. Shape-center system reference set to global cartesian system reference.
+    shape: [1, 3]
+    type: double
+
+  FILE_STL:
+    definition: Absolute path of a STL file
+    shape: [80]
+    type: string
+
+  STL_MODE:
+    definition: STL cutting process mode. Normal or negative cutting
+    shape: [1]
+    type: integer
+
+  STL_SCALING:
+    definition: Scaling parameter for components with STL shape
+    shape: [1]
+    type: double
+
+  STL_PADDING:
+    definition: Padding width in -X, +X, -Y, +Y, -Z and +Z directions for components with STL shape
+    shape: [6]
+    type: double
+
```

Comment:
There is a cuba called `FILE_STL` in this PR that defines a path for a STL file. The STL_MODE, STL_SCALING and STL_PADDING new CUBAS are nCad attributes, in concrete they are attributes of the nCad components that will have a STL shapes (this means, the shape they will have is defined by the triangles contained in the .stl file that make a 3D surface). They are not related to the .stl file indeed, they are just like any other attribute